app/models/insights_status.rb

```diff
@@ -0,0 +1,59 @@
+class InsightsStatus < HostStatus::Status
```

Comment:
Can we rename it to `InsightsClientReportStatus`. "Insights" is too broad term to understand what sort of status it will be.